gulpfile.babel.js

```diff
@@ -87,10 +87,14 @@ const AUTOPREFIXER_BROWSERS = [
 
 gulp.task('styles', ['css', 'sass']);
 gulp.task('css', () => {
+  let postcssOpts = {
+    autoprefixer: {
+      browsers: AUTOPREFIXER_BROWSERS
+    }
+  };
   let processors = [
-    autoprefixer(AUTOPREFIXER_BROWSERS),
     pcssnext,
```

Comment:
postcss-cssnext accepts some options (and it also have the same `browsers` option).

So this should do the trick

```js
pcssnext({ browsers: AUTOPREFIXER_BROWSERS }) // browsers option is propagated to autoprefixer
```